### PR TITLE
refactor(app): remove invalid type warnings for strings from atoms

### PR DESCRIPTION
**Pull request by koji**
# Overview
Add String() to remove warnings from atoms
The lines I added `String()` should be `string` since most of them are CSS property values for test cases(not null / not undefined).

This is part of RAUT-74

The number of warnings
```
edge: 3432
this branch: 3234
```

# Changelog
- Add String()

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/atoms/Banner/index.tsx [hunk 1]
```diff
@@ -119,7 +121,7 @@ export function Banner(props: BannerProps): JSX.Element {
       <Flex flex="1" alignItems={ALIGN_CENTER}>
         {props.children}
       </Flex>
-      {onCloseClick && !(isCloseActionLoading ?? false) ? (
+      {onCloseClick != null && !(isCloseActionLoading ?? false) ? (
```
Comment (jerader) [inline app/src/atoms/Banner/index.tsx]: can you maybe change `onCloseClick` prop on line 32 to say

`onCloseClick?: (() => void) | React.MouseEventHandler<HTMLButtonElement>`

Also, why is the `!= null` no longer needed?

File: app/src/atoms/Banner/index.tsx [hunk 2]
```diff
@@ -132,7 +134,7 @@ export function Banner(props: BannerProps): JSX.Element {
           )}
         </Btn>
       ) : null}
-      {isCloseActionLoading && (
+      {(isCloseActionLoading ?? false) && (
```
Comment (jerader) [inline app/src/atoms/Banner/index.tsx]: why do we not need the `?? false` here?

**Review by jerader — APPROVED** [review]
Nice Koji! thank you for removing a few hundred warnings! Just left a few questions.

File: app/src/atoms/Banner/index.tsx (hunk 1 above)
Comment (koji) [reply app/src/atoms/Banner/index.tsx]: I will update

File: app/src/atoms/Banner/index.tsx (hunk 2 above)
Comment (koji) [reply app/src/atoms/Banner/index.tsx]: I will take a look

**Comment by koji** [comment]
> Nice Koji! thank you for removing a few hundred warnings! Just left a few questions.

@jerader thank you for reviewing. I will address your feedback.

File: app/src/atoms/Banner/index.tsx (hunk 2 above)
Comment (koji) [reply app/src/atoms/Banner/index.tsx]: `isCloseActionLoading?: boolean`
maybe add `false` as the default value and keep the  `{isCloseActionLoading && (`

**Comment by koji** [comment]
- [x] resolve conflicts

**Pull request merged**